roles/openshift_node/tasks/install.yml

```diff
@@ -9,6 +9,7 @@
 - name: Install openshift packages
   package:
     name: "{{ l_node_packages | join(',') }}"
+    state: latest
```

Comment:
Perhaps make this a variable? That way during normal scaleup run it's just `present` but during an upgrade it's `latest`? I'm not sure how useful that would be so just throwing it out for consideration.

Reply:
I agree, this will help avoid issues where a specific version is required which can be installed manually prior to running the playbook. I'll submit an update to the PR to address.